R/vdiagram-data.R

```diff
@@ -63,21 +79,15 @@ historyMeasurementsLabel <- function(reportObject) {
   }
 }
 
+#' Set default historic flags
```

Comment:
I think this is to set the flag for if it's historic or not, while the other piece just creates the label. They're two different pieces of the metadata it looks like?